packages/remirror__core-utils/src/core-utils.ts

```diff
@@ -1227,6 +1227,39 @@ export function getDocument(): Document {
   );
 }
 
+/**
+ * Try to retrieve the window from the given HTML element or the global scope.
+ *
+ * @internal
+ */
+export function maybeGetWindow(
+  element?: Element | null,
+  document?: Document | null,
+): (Window & typeof globalThis) | null | undefined {
+  return (
+    element?.ownerDocument?.defaultView ??
+    document?.defaultView ??
+    (typeof window !== 'undefined' ? window : undefined) ??
+    getDomDocument()?.defaultView
+  );
+}
+
+/**
+ * @internal
+ */
+export function getWindow(
+  element?: Element | null,
+  document?: Document | null,
+): Window & typeof globalThis {
```

Comment:
Seems this is never called with both arguments.

Should this be two separate functions that wrap maybeGetWindow?

Reply:
Good catch. Done.